tests/fixtures/balsamic/TN_panel.hk

```diff
@@ -987,6 +987,58 @@
             ],
             "id": "TN_panel",
             "format": "vcf.gz.tbi"
+        },
+        {
+            "path": "TN_panel/analysis/umi/tumor_umi_consensusfiltered.merged.cram",
+            "path_index": [
+                "TN_panel/analysis/umi/tumor_umi_consensusfiltered.merged.cram.bai"
+            ],
+            "step": "mergeBam_tumor_umiconsensus",
+            "tag": [
+                "TN-panel",
+                "cram",
+                "umi-tumor-cram"
```

Comment:
```suggestion
                "umi-tumor-cram"
```
I have the tag as `umi_tumor` https://github.com/Clinical-Genomics/BALSAMIC/blob/e5f6c72547ae786a5feaca392b25b573eecd34ef/BALSAMIC/snakemake_rules/umi/mergetype_tumor_umi.rule#L20? should i change this tag name to `umi_tumor_cram` in balsamic?

Reply:
No, I believe that in BALSAMIC, when the tags are generated in `rule.py`, some additional ones are added to `umi_tumor`, such as the file type (cram) and the `case_id`. I will try to test it later generating the report with the balsamic command.